README.md

```diff
@@ -164,6 +164,24 @@ const appRoutes: Routes = [
 ]
 ```
 
+You can use `canLoad` to achieve lazy loading for not immediately necessary modules to keep the initial bundle size smaller.
```

Comment:
nit: not immediately loading?

Reply:
How about `for modules that are not immediately necessary`?